src/cli.lisp

```diff
@@ -44,8 +49,18 @@
        ,@body)))
 
 
-;;; Qi Install ($ qi --install [package] / $ qi -i [package]) internals
+;;; Qi install-deps ($ qi --install-deps project.asd)
+(defun opt-install-deps (input)
+  "Install the dependencies locally for the system definition file provided as INPUT."
+  (let* ((pwd (uiop:getenv "PWD"))
+         (asd-strings (cl-ppcre:split "\/" input))
```

Comment:
I _think_ there might be some other code that is already broken on Windows, but in any case I'm not currently worried about supporting it. Whatever works best here should be fine.